### PR TITLE
fix angular upgrade problem in composer

### PR DESCRIPTION
**Pull request by anthonyettinger**
@seanbrookes @jtary PTAL

### CODE REVIEW

File: client/www/scripts/modules/tracing/tracing.viz.module.js [hunk 1]
```diff
@@ -27993,7 +27993,8 @@ module.exports={
     "shasum": "58b1f9c9458653aee3ebd9b0a5bc1a79a47c6bd2",
     "tarball": "http://ci.strongloop.com:4873/strong-trace-waterfalltransform/-/strong-trace-waterfalltransform-1.0.18.tgz"
   },
-  "_resolved": "http://ci.strongloop.com:4873/strong-trace-waterfalltransform/-/strong-trace-waterfalltransform-1.0.18.tgz"
+  "_resolved": "http://ci.strongloop.com:4873/strong-trace-waterfalltransform/-/strong-trace-waterfalltransform-1.0.18.tgz",
+  "readme": "ERROR: No README data found!"
```
Comment (seanbrookes) [inline client/www/scripts/modules/tracing/tracing.viz.module.js]: is this intentional or a byproduct of the other changes?
you should revert this file is you didn't intend this update.

Comment (anthonyettinger) [inline client/www/scripts/modules/tracing/tracing.viz.module.js]: @seanbrookes I'm guessing it just happened when I ran the build. Should I revert?

**Comment by seanbrookes** [comment]
LGTM

**Comment by seanbrookes** [comment]
LGTM

**Pull request merged**